wiki/en/Running-a-Server.md

```diff
@@ -274,7 +274,7 @@ For macOS, start a Terminal window and run Jamulus with the desired options like
 
 ## Running an Unregistered Server
 
-It is highly recommended to test your Server on a **Public Directory** first so as to narrow down any subsequent problems in unregistered mode.
+It is highly recommended to test your Server by listing it on one of the built-in Directories **first** so as to narrow down any subsequent problems in unregistered mode.
```

Comment:
listing vs registering is also not consistent. I think we talk about registering in other places, so why not here too?

Reply:
Probably registering in that case, yes.